src/DataDepsGenerators.jl

```diff
@@ -126,15 +119,17 @@ function combine_all(values::Vector, sym::Symbol)
     ret
 end
 
-combine(::Missing, ::Missing, z) = missing
-combine(::Missing, x, z) = x
-combine(x, ::Missing, z) = x
-combine(x::Vector,y::Vector, z) = length(x) > length(y) ? x : y
-combine(x::String,y::String,s::Symbol) = combine(x,y,Val{s}())
-combine(x::String,y::String, ::Val{:license}) = length(x) < length(y) ? x : y
-combine(x::String,y::String, z) = length(x) > length(y) ? x : y
-combine(x::Union{DateTime, Date},y::String, z) = x
-combine(x::String,y::Union{DateTime, Date}, z) = y
+#Redispatch based on Value
+combine(x::String, y::String, s::Symbol) = combine(x,y,Val{s}())
```

Comment:
Did 
```
combine(x::T1, y::T2, s::Symbol) where {T1,T2} = combine(x,y,Val{s}())
```
not work out?

Reply:
Nope, it didn't, cause it was ambigous with `combine(x::String, y::String, ::Any)` and others.

Reply:
hmm might want to ask on #helpdesk or discourse about avoiding that.